.vscode/launch.json

```diff
@@ -91,10 +95,17 @@
       "args": [
         "--extensionDevelopmentPath=${workspaceRoot}/extensions/ql-vscode",
         "--extensionTestsPath=${workspaceRoot}/extensions/ql-vscode/out/vscode-tests/cli-integration/index",
+        "--disable-gpu",
+        "--disable-extension",
+        "eamodio.gitlens",
+        "--disable-extension",
+        "github.codespaces",
+        "--disable-extension",
+        "github.copilot",
         "${workspaceRoot}/extensions/ql-vscode/src/vscode-tests/cli-integration/data",
-        // Add a path to a checked out instance of the codeql repository so the libraries are
+        // Uncomment the last line and modify the path to a checked out 
+        // instance of the codeql repository so the libraries are
         // available in the workspace for the tests.
-        // "${workspaceRoot}/../codeql"
```

Comment:
I think you should keep this line as an example of what the argument should look like. If you have `github/codeql` checked out as a sibling repo, the commented line should not need to be changed.

Reply:
Oops, you're right! I left that line in the commit on accident.